.github/workflows/rusk_build.yml

```diff
@@ -0,0 +1,65 @@
+name: Compile Rusk Binaries
+
+on:
+  workflow_dispatch:
+    inputs:
+      dusk_blockchain_ref:
+        description: "Git branch, ref, or SHA to checkout"
+        required: true
+        default: "master"
+
+jobs:
+  build_and_publish:
+    name: Build Rusk binaries for ${{ matrix.os }} (${{ matrix.features }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-24.04, macos-12, arm-linux]
+        compiler: [cargo]
+        features: [default, archive]
+        include:
+          - os: ubuntu-24.04
```

Comment:
Are we moving support from 22.04 to 24.04?

Reply:
@herr-seppia we don't explicitly mention 22.04 or 24.04 anywhere in our Rusk repo. I've been testing the binaries on 22.04, 24.04 and 24.10, and they all work fine on my end.

In my opinion it makes sense to explicitly state we support 24.04, given it has been out for 8 months now and "End of Standard Support" runs to June 2029. But I don't have a very strong opinion here. What do you think?